services/datamanager/builtin/capture_test.go

```diff
@@ -201,6 +213,19 @@ func TestDataCaptureEnabled(t *testing.T) {
 					test.That(t, oldCaptureDirFiles[i].Size(), test.ShouldEqual, oldCaptureDirFilesAfterWait[i].Size())
 				}
 			}
+			cancelPassTime2()
+			<-donePassingTime2
 		})
 	}
 }
+
+func waitForCaptureFiles(captureDir string) {
+	files := getAllFileInfos(captureDir)
+	for i := 0; i < 200; i++ {
```

Comment:
[superdupernit] To make it clear why we do this loop, can we name/extract out 200 into a test const?